tools/tf_env_collect.sh

```diff
@@ -142,35 +159,33 @@ LD_DEBUG=libs ${python_bin_path} -c "import tensorflow"  2>>${OUTPUT_FILE} > /tm
   fi
   
   
-  echo
-  echo '== nvidia-smi ==================================================='
+  print_header nvidia-smi
   nvidia-smi 2>&1
   
-  echo
-  echo '== cuda libs  ==================================================='
-} >> ${OUTPUT_FILE}
+  print_header 'cuda libs'
 
-find /usr/local -type f -name 'libcudart*'  2>/dev/null | grep cuda |  grep -v "\\.cache" >> ${OUTPUT_FILE}
```

Comment:
These two lines did two full disk searches through the drive for these files, this find will always contain "cuda" since the file is "libcudart" so I assumed that they wanted to find "cuda" the directory. The second pass was looking for "libudnn"? Genuinely unsure what they were going for here.

This has been updated to be a single loop through the drive, and look for "libcudart and "libcudnn".